WordPress/Jetpack/Classes/JetpackAuthenticationManager.swift

```diff
@@ -48,14 +48,19 @@ struct JetpackAuthenticationManager: AuthenticationHandler {
         onCompletion(authenticationResult)
     }
 
-    func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void) -> Bool {
-        if hasJetpackSites() {
+    func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, onDismiss: @escaping () -> Void, windowManager: WindowManager) -> Bool {
```

Comment:
Super tiny nit, what do you think of making the onDismiss closure the final argument instead of windowManager?
```suggestion
    func presentLoginEpilogue(in navigationController: UINavigationController, for credentials: AuthenticatorCredentials, windowManager: WindowManager, onDismiss: @escaping () -> Void) -> Bool {
```

Reply:
Done! 12ff62a82dd3c3859869d6742fc954daa9dae3f7